app/controllers/articles_controller.rb

```diff
@@ -7,14 +7,16 @@ def index
   end
 
   def show
+    @article = Article.find(params[:id])
+    @user = @article.user
```

Comment:
This is not needed as you can use `@article.user.username`

Reply:
Done